tasks/deploy/config/mainnet-fork/factory.ts

```diff
@@ -0,0 +1,60 @@
+import { Address, parseEther } from "viem";
+import { HyperdriveFactoryConfig, parseDuration } from "../../lib";
+
+export const MAINNET_FORK_FACTORY: HyperdriveFactoryConfig = {
+    name: "FACTORY",
+    prepare: async (hre, options) => {
+        await hre.hyperdriveDeploy.ensureDeployed(
+            "FACTORY_FORWARDER",
+            "ERC20ForwarderFactory",
+            [],
+            options,
+        );
+    },
+    constructorArguments: async (hre) => [
+        {
+            governance: process.env.ADMIN! as `0x${string}`,
+            deployerCoordinatorManager: process.env.ADMIN! as `0x${string}`,
+            hyperdriveGovernance: "0xc187a246Ee5A4Fe4395a8f6C0f9F2AA3A5a06e9b",
+            defaultPausers: [
+                (await hre.getNamedAccounts())["deployer"] as Address,
+            ],
```

Comment:
Nice. Can we make sure this gets updated in the sepolia scripts too if it hasn't already? We should also add governance here just in case.